src/facade/dragonfly_connection.h

```diff
@@ -60,22 +63,21 @@ class Connection : public util::Connection {
   using BreakerCb = std::function<void(uint32_t)>;
   using ShutdownCb = std::function<void()>;
   using ShutdownHandle = unsigned;
+  using MonitorMessage = std::string;
 
   // PubSub message, either incoming message for active subscription or reply for new subscription.
   struct PubMessage {
-    std::string pattern{};            // non-empty for pattern subscriber
-    std::shared_ptr<char[]> buf;      // stores channel name and message
-    size_t channel_len, message_len;  // lengths in buf
+    std::string pattern{};                 // non-empty for pattern subscriber
+    std::shared_ptr<char[]> buf;           // stores channel name and message
+    std::size_t channel_len, message_len;  // lengths in buf
```

Comment:
we usually do not use std::size_t, i do not see much value in adding this prefix if we already have it in the global space.

Reply:
In this particular case I was trying to match the style of the surrounding code since all of the other types used from the standard library do include the namespace (string, shared_ptr, variant, etc). I'll revert if you'd prefer not to include the namespace.